release_management/_utils.py

```diff
@@ -38,8 +38,17 @@ class ReleaseVersion:
     """
 
     def __init__(self, base, suffix):
+        """
+        Initializer.
+        :param str base: Base version
+        :param str or NoneType: Version suffix
```

Comment:
With a complex type like ```str or NoneType``` best to split into two:

```
:param suffix: version suffix
:type suffix: str or NoneType
```

you'll see examples elsewhere.

Reply:
Resolved.